dali/operators/image/color/debayer_cpu.cc

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2022-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <opencv2/imgproc.hpp>
+
+#include "dali/kernels/imgproc/color_manipulation/debayer/debayer.h"
+#include "dali/operators/image/color/debayer.h"
+#include "dali/util/ocv.h"
+
+namespace dali {
+
+using namespace dali::kernels::debayer;
+
+[[nodiscard]] cv::ColorConversionCodes toOpenCVColorConversionCode(DALIBayerPattern pattern,
```

Comment:
`[[nodiscard]]` is intended for safety - e.g. to force error handling or to avoid resource leaks. Discarding a result of a plain function like this one doesn't have any safety implications.